lib/nexpose/global_blackout.rb

```diff
@@ -0,0 +1,43 @@
+module Nexpose
+
+  class GlobalBlackout < APIObject
+    require 'json'
+    include JsonSerializer
+
+    # [Array] Blackout starting dates, times and duration for blackout periods.
+    attr_accessor :blackout
+
+    def initialize(blackout)
+      @global_blackout = Array(blackout)
```

Comment:
should this just be ``@blackout`` or do we need to add an ``attr_`` for ``global_blackout`` as well? 
I might just be missing something due to a case of the Mondays.